src/fourlang/dep_to_4lang.py

```diff
@@ -244,6 +247,13 @@ def create_from_line(line):
             # logging.info('adding new rel from: {0}'.format(dep))
             rel = dep.split('_', 1)[1].upper()
 
+        # Universal Dependencies
+        if ((dep.startswith('acl:') and not dep.startswith('acl:relcl')) or
+                dep.startswith('advcl:') or
+                dep.startswith('nmod:')) and rel is None:
+            logging.info('adding new rel from: {0}'.format(dep))
+            rel = dep.split(':', 1)[1].upper()
```

Comment:
E.g.:
word: absenteeism
deps: 
"nmod:without(absence-2, reason-10)" => absenteeism <1- WITHOUT -2> reason
"nmod:from(absence-2, school-6)" => absenteeism <1- FROM -2> school

is it ok this way, or should I add comments into the code?